### PR TITLE
JBPM-5424: Version selector does not work in the new form modeler.

### PR DESCRIPTION
**Pull request by pefernan**
Merge with: https://github.com/droolsjbpm/kie-wb-common/pull/646

### CODE REVIEW

**Comment by pefernan** [comment]
@ederign can you take a look please?

**Comment by pefernan** [comment]
jenkins retest this.

File: uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/test/java/org/uberfire/ext/layout/editor/client/LayoutEditorPresenterTest.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
+package org.uberfire.ext.layout.editor.client;
```
Comment (manstis) [inline uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/test/java/org/uberfire/ext/layout/editor/client/LayoutEditorPresenterTest.java]: Late to the party.. but this should probably have a license.

File: uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/test/java/org/uberfire/ext/layout/editor/client/test/TestLayoutEditorPresenter.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package org.uberfire.ext.layout.editor.client.test;
```
Comment (manstis) [inline uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/test/java/org/uberfire/ext/layout/editor/client/test/TestLayoutEditorPresenter.java]: License.

Comment (pefernan) [inline uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/test/java/org/uberfire/ext/layout/editor/client/test/TestLayoutEditorPresenter.java]: @manstis I hate when Intellij doesn't store the license settings between projects. I'll send a new PR with them.

Comment (pefernan) [inline uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/test/java/org/uberfire/ext/layout/editor/client/test/TestLayoutEditorPresenter.java]: @manstis https://github.com/uberfire/uberfire/pull/615

Comment (psiroky) [inline uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/test/java/org/uberfire/ext/layout/editor/client/test/TestLayoutEditorPresenter.java]: @pefernan you can configure this globally, for all projects.

**Pull request merged**